reference-implementation/to-upstream-wpts/writable-streams/write.js

```diff
@@ -197,3 +197,15 @@ promise_test(() => {
           assert_equals(writeCount, numberOfWrites, `should have called sink's write ${numberOfWrites} times`));
   });
 }, 'a large queue of writes should be processed completely');
+
+promise_test(() => {
+  const stream = recordingWritableStream();
+  const w = stream.getWriter();
+  const WritableStreamDefaultWriter = w.constructor;
+  w.releaseLock();
+  const writer = new WritableStreamDefaultWriter(stream);
```

Comment:
Okay. So, we should keep in mind that invocation of `.getWriter()` shouldn't be used for detecting obtaining of a writer. We've already switched to not use public method invocation for controlling its behavior for pipeTo() implementation. So, yes, I understand it's ok.